packages/@aws-cdk/aws-stepfunctions-tasks/lib/sagemaker/create-endpoint-config.ts

```diff
@@ -0,0 +1,135 @@
+import * as iam from '@aws-cdk/aws-iam';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import * as cdk from '@aws-cdk/core';
+import { integrationResourceArn, validatePatternSupported } from '../private/task-utils';
+import { ProductionVariant } from './base-types';
+
+/**
+ * Properties for creating an Amazon SageMaker endpoint configuration
+ *
+ * @experimental
+ */
```

Comment:
include a doc link to the step functions connectors since it could be handy for users and gives a jump off point for more information (https://docs.aws.amazon.com/step-functions/latest/dg/connect-sagemaker.html)